pytest_reportportal/service.py

```diff
@@ -284,23 +284,24 @@ def start_pytest_item(self, test_item=None):
         if self.rp is None:
             return
 
+        parent_item_id = None
         for part in self._item_parts[test_item]:
             if self._hier_parts[part]["start_flag"]:
-                self.parent_item_id = self._hier_parts[part]["item_id"]
+                parent_item_id = self._hier_parts[part]["item_id"]
                 continue
             self._hier_parts[part]["start_flag"] = True
             payload = {
                 'name': self._get_item_name(part),
                 'description': self._get_item_description(part),
                 'start_time': timestamp(),
                 'item_type': 'SUITE',
-                'parent_item_id': self.parent_item_id,
+                'parent_item_id': parent_item_id or self.parent_item_id,
                 'code_ref': str(test_item.fspath)
             }
             log.debug('ReportPortal - Start Suite: request_body=%s', payload)
             item_id = self.rp.start_test_item(**payload)
```

Comment:
Rather than use `item_id` here, you could use `parent_item_id`. This would resolve two issues:

1. You wouldn't have to explicitly set `parent_item_id` in line 304
2. You wouldn't have a variable name conflict for `item_id`, like you do now with this line and line 323